src/client/pages/ConnectPayment/sections/ConnectPayment.tsx

```diff
@@ -133,6 +131,14 @@ const InstructionItemSeparator = styled.div`
   height: 0.5rem;
 `
 
+const getPaymentAlreadyConnectedUrl = ({
```

Comment:
not sure if this is correct. Adyen is the one used in DK and NO, in SE we need to use Trustly so changes should be done there. Car is still only available in SE right?

Reply:
Car is only available for Sweden, yes, but the functionality should be correct for all markets - if payment is already connected there's no need to show the "Connect payment page".. or?

The GraphQL endpoint returns `"ACTIVE`". 

Fredrik Areschoug wrote:
> So many threads so little time. But yeah use `payinMethodStatus` which takes care of both adyen and trustly :)

Reply:
@dendoesit please get back to me on the above, I wouldn't want to break anything and you know more about payments than I do 🙃